learn/wavemaker-release-notes/v10-7-0.md

```diff
@@ -65,6 +68,14 @@ Introduced the following methods to get default options provided for the Richtex
 - `getLib()`: Returns the library used for the widget.
 - `overrideDefaults(options)`: Customize the options which the library supports.
 
+### Backend Enhancements
+
+- For Rest/Swagger services, the rest call goes from the backend when you enable proxy, making the direct path empty in service definitions.
```

Comment:
i didn't understand this. Does not pass the "so, what" test

Reply:
@prakashbuthukuri, do you know what problem it solves?

Reply:
If we didnt make the direct path as empty, then the full service path is visible in the network/developer console
(full path -> host name, request path and any path or query parameters configured for that service). 
Some customers raised security concern over exposing the direct path when proxy is enabled. 
So to address that we make the direct path as empty when proxy is enabled.

Reply:
Lets remove this line @swethakundaram